scripts/winresources/event_messages.mc

```diff
@@ -0,0 +1,39 @@
+MessageId=1
```

Comment:
Ok, I will remove this file and the file from the generation script which uses it.